### DISCUSSION TITLE
False Virus Flag?

### DISCUSSION THREAD

**Discussion by waltersd20** [SamDel/ChromeCast-Desktop-Audio-Streamer#129] [Q&A]
I'm a bit of a nut, but basically run any new installs/updates through VirusTotal. v3.8 reported clean but 3.9 has Bkav reporting an ASP WebShell exploit. Was the installer improperly configured? Thanks!
![image](https://user-images.githubusercontent.com/99553439/153721310-9746bc72-fd91-4426-b40e-222b8493f3e2.png)

**Comment by SamDel** [comment]
I don't know why VirusTotal thinks there's a virus in the installer, probably it's a false alert. If you want to be safe you better skip this release, or download the code and compile it yourself.
